### ISSUE TITLE
New 2.6 version

### ISSUE THREAD

**Issue by mmoreram** [doctrine/common#745]
Last 19 august an issue was merged to provide support for PHP 7.1 null returns, but is not available under any tag yet, and this feature is a must if we want to use 7.1 (like in my case); otherwise it doesn't work due to class compatibility.

Can you tag a new version please? That would be awesome :)

Thanks!

**Comment by Ocramius** [comment]
@mmoreram 7.1 is not yet released, and it changed quite a while over the last month, especially in reflection-specific API.

It will be released AFTER 7.1 went stable.

**Comment by mmoreram** [comment]
So... what's the point? I'm not asking anyone to give me support with my project, but to release a version with commits that you merged into master, to make sure this issue is not a block for anyone...

**Comment by Ocramius** [comment]
@mmoreram only after 7.1.0 is stable. I already had to directly go to php-src and communicate some BC breaks when we were (and we still are) in RC phase. If further BC breaks appear, I don't want people to run around like crazy fixing doctrine/common because a release is not compatible.

Until 7.1.0 is out, no 2.6 release. You can use `"doctrine/common": "dev-master@DEV as 2.6.99"` until then.

**Comment by mmoreram** [comment]
@Ocramius yes, that's what I'm trying, but any version of the ORM is only compatible with `<2.7-dev`, and dev-master is aliased with `<2.7-dev`... so yeah... not sure how I will manage it without doing a useless fork...

**Comment by Ocramius** [comment]
Wait few weeks? Or alias manually, as I already suggested.
